requirements.txt

```diff
@@ -2,3 +2,4 @@ intake>=0.5.1
 intake-xarray>=0.3
 scikit-image>=0.15
 sat-stac>=0.1.3
+rasterio
```

Comment:
I don't think this needs to be a hard dependency. I think users will often want rasterio but it should be possible to use intake-stac without it.